Sources/SQLKit/Query/SQLUnion.swift

```diff
@@ -35,6 +47,18 @@ public struct SQLUnion: SQLExpression {
                 appendQuery(query)
             }
         }
+        if !self.orderBys.isEmpty {
+            serializer.write(" ORDER BY ")
+            SQLList(self.orderBys).serialize(to: &serializer)
+        }
+        if let limit = self.limit {
+            serializer.write(" LIMIT ")
+            serializer.write(limit.description)
+        }
+        if let offset = self.offset {
+            serializer.write(" OFFSET ")
+            serializer.write(offset.description)
+        }
```

Comment:
Whoops, I missed this part before- Can you move these writes into the `serializer.statement {` closure and update them as per below? GitHub doesn't let me apply a code suggestion to lines that weren't changed in the diff, so I can't quite show it cleanly. (Also, someone should really fix `SQLSelect`'s serialization to at least properly use a numeric literal expression, but that's out of scope for this PR.)
```suggestion
            if !self.orderBys.isEmpty {
                statement.append("ORDER BY")
                statement.append(SQLList(self.orderBys))
            }
            if let limit = self.limit {
                statement.append("LIMIT")
                statement.append(SQLLiteral.numeric("\(limit)"))
            }
            if let offset = self.offset {
                statement.append("OFFSET")
                statement.append(SQLLiteral.numeric("\(offset)"))
            }
```

Reply:
Oh totally, no problem. Yeah that makes sense with SQLSelect, since I definitely just based this code off that one.